Makefile

```diff
@@ -199,6 +199,12 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+list-publishable-modules:
+	./bin/list-publishable-modules.js
+
+publish-module:
+	( cd $(path) && make publish )
```

Comment:
Fixed in: https://github.com/belcherj/wp-calypso/commit/7e4530096e1173dbd13006fde69e4ad6f456832a